pkgs/test_core/lib/src/util/package_config.dart

```diff
@@ -18,3 +20,15 @@ final Future<Uri> packageConfigUri = () async {
   }
   return uri;
 }();
+
+final _originalWorkingDirectory = Directory.current.uri;
```

Comment:
I am seeing some breakage internally, I think the issue is that for precompiled tests we need to use the precompiled dir instead of the current dir?

Reply:
Ok, so the issue is actually not that, the this logic is working correctly to convert a URI to a package: URI, but the test they are trying to launch actually lives in another package. It is getting converted to a package: uri (as it should) and then failing to load for some reason.